src/redux/slices/AudioPlayer/state.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
need to breakdown this file into separate line, but will do it if this PR is approved.